### ISSUE TITLE
Use a button to trigger routines

### ISSUE THREAD

**Issue by JxnathanDE** [kakopappa/sinric#333]
Hi,

firstly, thanks for the great tool you created.
My suggestion is a button like the "Echo Buttons", so that I can, for example, use it as a button connected to an ESP digital PIN to turn a relay from an (other) ESP on and update the status in the app.

**Comment by kakopappa** [comment]
you can send a UDP command from one esp to another esp?

**Comment by JxnathanDE** [comment]
Yes, but if I want to send an Alexa command to turn another device off/on, which isn't running over an ESP. Also, the status in the Alexa app isn't updated with a command from ESP to ESP.
